### DISCUSSION TITLE
g silver sticker config

### DISCUSSION THREAD

**Discussion by Biggingus** [aluminum-ice/pwnagotchi#121] [Q&A] [answered]
last post says that it has support for the  "e-Ink Waveshare 2.13" model G (silver Sticker)"  what do you need to put into the  "config.toml" in order for it to pick up the display tried ui.display.type = "waveshare_G" and also waveshare_3 but the screen has not come to life  
thank you

**Comment by wpa-2** [comment, marked as answer]
Never mind try this 
waveshare213g

https://github.com/aluminum-ice/pwnagotchi/commit/2437f5e11c84504c5634b5ccd85cf2b0ae205f48

**Reply by Biggingus** [reply]
Thank you so much
